unime/src/i18n/nl-NL/index.ts

```diff
@@ -199,6 +205,30 @@ Door onze Service te gebruiken, erken je dat je deze Voorwaarden heeft gelezen,
         TITLE_2: 'van je app',
         SUBTITLE: 'Ben jij meer een nachtuil?',
       },
+      SECURITY: {
+        LABEL: 'Beveiliging',
+        NAVBAR_TITLE: 'Beveiliging',
+        SWITCH_LABEL: 'Ontgrendelen met {type}',
+        BIOMETRIC_TYPE: {
+          ANDROID: {
+            FACE_ID: 'gezichtsherkenning',
+            TOUCH_ID: 'vingerafdruk',
+          },
+          IOS: {
+            FACE_ID: 'Face ID',
+            TOUCH_ID: 'Touch ID',
+          },
+          GENERIC: 'biometrie',
+        },
+        ENABLE: {
+          DIALOG_TITLE: '{type} inschakelen',
```

Comment:
`Inloggen met {type} inschakelen`?

Or in case you want to keep it the way it is, then make sure to capitalize:
* `FACE_ID: 'gezichtsherkenning'` -> `FACE_ID: 'Gezichtsherkenning'`
* `TOUCH_ID: 'vingerafdruk'` -> `FACE_ID: 'Vingerafdruk'`

The latter option would cause problems though in `DIALOG_CONTENT: 'Voer je wachtwoord in om {type} in te schakelen'` and `DIALOG_CONTENT: 'Voer je wachtwoord in om {type} uit te schakelen'` I suppose since `{type}` is in the middle of those sentences..

Reply:
It is actually capitalize in the frontend due to that very specific issue ... So only on one page, the word is capitalized and in others it's not. I know this is not ideal, but we'll eventually move to a different i18n library anyway which hopefully solves the issue.